rules/mid.lua

```diff
@@ -14,7 +14,7 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ]]--
-
+local util = require "rspamd_util"
```

Comment:
I would suggest to name variable as `rspamd_util` not `util` to avoid further ambiguity with `lua_util`.